interface/ceed-elemrestriction.c

```diff
@@ -146,6 +146,22 @@ int CeedElemRestrictionRestoreOffsets(CeedElemRestriction rstr,
   return 0;
 }
 
+/**
+  @brief Get the strided status of a CeedElemRestriction
+
+  @param rstr             CeedElemRestriction
+  @param[out] status      Variable to store strided status, 1 if strided else 0
+
+  @return An error code: 0 - success, otherwise - failure
+
+  @ref Backend
+**/
+int CeedElemRestrictionGetStridedStatus(CeedElemRestriction rstr,
+    bool *status) {
+  *status = rstr->strides ? 1 : 0;
+  return 0;
+}
+
```

Comment:
@nbeams, is this what you were thinking of?

Reply:
Yes, thanks!  I think this will be very useful